doc_indexer/.vscode/launch.json

```diff
@@ -10,7 +10,18 @@
       "request": "launch",
```

Comment:
Should we be committing `.vscode` dir?

Reply:
I am using vs code more often than Pycharm. Tanweer is also using VS code. It might be useful for him too. 
I am just sharing the some useful run configuration so that he just can use it.